plugin/core/windows.py

```diff
@@ -295,12 +296,13 @@ def start_async(self, config: ClientConfig, initiating_view: sublime.View) -> No
                 "Re-enable by running \"LSP: Enable Language Server In Project\" from the Command Palette.",
                 "\n\n--- Error: ---\n{1}"
             )).format(config.name, str(e))
+            status = f"LSP: Failed to start {config.name}… See console"
```

Comment:
Do we need the `LSP:` prefix in the status messages? It should be obvious if there is either the `config.name` or when it was something directly triggered by the user.

Reply:
ok, there is also the bigger console message with more detailed errors to fall back to